File: src/main/java/com/uiptv/util/LogUtil.java
package com.uiptv.util;
import java.util.Map;

public class LogUtil {
    private LogUtil() {
    }

    public static void httpLog(String url, HttpUtil.HttpResult response, Map<String, String> params) {
        com.uiptv.util.AppLog.addInfoLog(LogUtil.class, HttpUtil.formatHttpLog(url, response, params));
    }
}

File: server/src/main/kotlin/com/uiptv/util/LogUtil.kt
package com.uiptv.util

object LogUtil {
    @JvmStatic
    fun httpLog(url: String, response: HttpUtil.HttpResult, params: Map<String, String>) {
        AppLog.addInfoLog(LogUtil::class.java, HttpUtil.formatHttpLog(url, response, params))
    }
}
